lib/recurly/unique_coupon_list.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class Recurly_UniqueCouponList extends Recurly_Pager
+{
+
+  public static function createFromUri($uri, $client) {
+    $path = str_replace('/v2', '', parse_url($uri, PHP_URL_PATH));
+    $query = parse_url($uri, PHP_URL_QUERY);
+    $params = parse_str($query);
+
+    $list = new self(null, $client);
+    $list->_loadFrom($path, $params);
```

Comment:
Okay yeah with the new pagination changes we don't need the _loadFrom you can just pass the full URL into the constructor.